stacks-core/create-source-binary/README.md

```diff
@@ -10,6 +10,7 @@ Builds a binary for the given architecture and uploads it to artifacts.
 | ---------------- | ----------------------------- | -------- | ------------------------ |
 | `arch`           | Binary's build architecture   | `true`   | null                     |
 | `tag`            | The tag for the release       | `false`  | null                     |
```

Comment:
Here, the tag requirement needs to be updated to `true` (since it was updated in `action.yml`)

Reply:
done